cfn-resources/private-endpoint-aws/upgradeguidev2/V2-UpgradeGuide.md

```diff
@@ -0,0 +1,620 @@
+# V2 Private Endpoint Upgrade Guide
```

Comment:
As mentioned in other comment, given this is relevant for users using existing PrivateEndpoint resource, would place migration guide there.